### PR TITLE
Icon guidance

### PR DESCRIPTION
**Pull request by davidhunter08**
- added icon guidance with examples
- removed icons that are not used
- created `svg` file for each icon
- added `aria-hidden="true` and `focusable="false"` to each icon
- refactored icons macro to include `svg` files

### CODE REVIEW

**Comment by davidhunter08** [comment]
Messages with unread status icons need updating as they don't currently have the red dot on them.

Screenshot 2024-07-19 at 17 52 26

https://www.figma.com/design/6f2CbcZ7cnpNrtKEcfQp8X/NHS-App-Design-System?node-id=1295-1786&t=XJrYxovldYo5IbsQ-1

Screenshot 2024-07-19 at 17 55 48

File: src/styles/icon/_icons.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.nhsapp-icon {
```
Comment (edwardscull) [inline src/styles/icon/_icons.scss:1]:
```sass
.nhsapp-icon {
 // styles
 &__unread-indicator {
  // styles
 }
}
```
The `&` can make this a little cleaner meaning it's easier to add/extend in the future.

**Review by edwardscull — CHANGES_REQUESTED** [review]

File: src/all.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
 @import "node_modules/nhsuk-frontend/packages/core/settings/all";
 
 // styles
+@import "styles/icon/icons";
```
[the remark this answers is not in the document]
Comment (davidhunter08) [reply src/all.scss:5]: I was following what we already had for the other components / styles. 

For example: https://github.com/nhsuk/nhsapp-frontend/blob/main/src/components/badge/_badge.scss

Do you recommend changing them all to `_index.scss`?

[the remark this answers is not in the document]
Comment (edwardscull) [reply src/all.scss:5]: I think it's a good thing to make consistent yeh, but happy for it not be changed in this PR

**Pull request merged**